lib/utils.js

```diff
@@ -219,6 +219,12 @@ async function parseLocalizableStrings (opts) {
 
   const resultStrings = {};
   for (const resourcePath of resourcePaths) {
+    if (!util.isSubPath(
+      path.isAbsolute(resourcePath) ? resourcePath : path.resolve(process.cwd(), resourcePath),
```

Comment:
This seems like it could be extracted out. If not into `appium-support`, into a little function here.

Reply:
yep, changed